cfgov/jinja2/v1/ask-cfpb/landing-page.html

```diff
@@ -48,25 +57,49 @@ <h2 class="u-show-on-mobile u-mt30">Browse questions by category</h2>
 
                                             <h2>
+                                                {% if page.language == 'es' %}
+                                                <a href="{{ slugurl( 'categoria-' + category.slug_es ) }}">
+                                                {% else %}
                                                 <a href="{{ slugurl( 'category-' + category.slug ) }}">
+                                                {% endif %}
                                                 {{ category_icon.render( category.name ) }}
+                                                {% if page.language == 'es' %}
+                                                {{ category.name_es }}
+                                                {% else %}
                                                 {{ category.name }}
+                                                {% endif %}
                                                 </a>
                                             </h2>
 
                                             <ul class="m-list question_list m-list__flush-left u-mt10 u-mb30">
-                                                {% for a in category.featured_answers('en') %}
-                                                <li class="m-list_item">
-                                                {{ partially_styled_link.render( {'plain_text': a.question, 'underlined_text': 'Read answer', 'url': a.url} ) }}
-                                                </li>
-                                                {% endfor %}
+                                                {% if page.language == 'es'%}
+                                                    {% for a in category.featured_answers('es') %}
+                                                    <li class="m-list_item">
+                                                    {{ partially_styled_link.render( {'plain_text': a.question, 'underlined_text': 'Leer la respuesta', 'url': a.url} ) }}
+                                                    </li>
+                                                    {% endfor %}
+                                                {% else %}
+                                                    {% for a in category.featured_answers('en') %}
+                                                    <li class="m-list_item">
+                                                    {{ partially_styled_link.render( {'plain_text': a.question, 'underlined_text': 'Read answer', 'url': a.url} ) }}
+                                                    </li>
+                                                    {% endfor %}
+                                                {% endif %}
+
                                             </ul>
 
                                             <div class="content-l_footer__equal-height">
                                                 <p class="short-desc u-mb0">
```

Comment:
`u-mb0` could possibly be `block block__flush-bottom block__flush-top` if we want to standardize on block.